lib/VCard.ts

```diff
@@ -286,6 +286,16 @@ ${lastName};${firstName};${additional};${prefix};${suffix}\
     return this
   }
 
+  /**
+   * Add nickanem
+   *
+   * @param {string} nickname
+   */
+  public addNickname(nickname: string): this {
```

Comment:
**suggestion:** per [RFC 2426, section 3.1.3](https://www.rfc-editor.org/rfc/rfc2426#section-3.1.3), the `nickname` property can support comma-separated values. what do you think of adding support for both `string` and `string[]` types?